### DISCUSSION TITLE
stargate variant symbol color

### DISCUSSION THREAD

**Discussion by electikos** [Povstalec/StargateJourney#147] [Q&A]
I know how to make a variant with a texture but I can't figure it out how function the new RGB system use for the variants.
(or I make some thing wrong)

my :
`"symbol_color": { "red": 150, "green": 150, "blue": 0, "alpha": 1.0 },
 "encoded_symbol_color": { "red": 255, "green": 200, "blue": 0, "alpha": 1.0 },
"engaged_symbol_color": { "red": 255, "green": 255, "blue": 0, "alpha": 1.0 },`

atlantis:
`"symbol_color": { "red": 0.0, "green": 0.392, "blue": 0.784, "alpha": 1.0 },
"encoded_symbol_color": { "red": 0.0, "green": 0.784, "blue": 1.0, "alpha": 1.0 },
"engaged_symbol_color": { "red": 0.0, "green": 0.784, "blue": 1.0, "alpha": 1.0 },`

**Comment by Povstalec** [comment]
Can you describe *what* exactly is wrong? This doesn't tell me much about what your issue actually is, so I can't really help
